### DISCUSSION TITLE
Draft issues disappeared from project in bulk

### DISCUSSION THREAD

**Discussion by rmshaffer** [community/community#67638] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

We have a GitHub project here: https://github.com/orgs/amazon-braket/projects/2/views/6

We had about ~50 draft issues on this board. About 10 minutes ago they all (except one) disappeared completely. Not sure if this was due to some inadvertent button click or keypress, but there was no indication in the UX of what happened. The missing draft issues do not appear in Archived items or anywhere else. Can we get them back?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by kushan-uber** [comment]
same here, all of my issues except one were removed from the projects. All drafts were deleted.

**Reply by leander-hass** [reply]
Yeah same here, based on the Date on the remaining draft-issue it could be the very first I created, but not sure. My project is private.

**Comment by ilyabo** [comment]
Same with me in these two projects (private):
https://github.com/users/ilyabo/projects/2/views/1
https://github.com/users/ilyabo/projects/1/views/4

One of them was for collecting ideas on the project. I've been doing it for several years. If they are irreversibly lost, it's a disaster for me.

**Comment by MlanzAgea** [comment]
Same here
All the drafts that we had with my team now are gone and only one remains

With this projects
https://github.com/orgs/Tecnologia-Comercial-Agea/projects/24/views/1
https://github.com/orgs/Tecnologia-Comercial-Agea/projects/53/views/1

**Comment by papamoamac** [comment]
same here - all drafts are gone. WTF!!!

**Reply by papamoamac** [reply]
I am fairly new to github. Anyone got any idea of how to ticket this with support? I am just on free account.

**Reply by papamoamac** [reply]
Everything just returned :-) !!!!

**Comment by ViniciusBatestin** [comment]
Same here all drafts gone :/  (private) 😭

**Comment by vitormmx** [comment]
same here!  I will burn my house down!

**Reply by ViniciusBatestin** [reply]
😂

**Comment by rmshaffer** [comment]
Issue seems to be resolved now. All of the draft issues have returned: https://github.com/orgs/amazon-braket/projects/2/views/6

**Reply by leander-hass** [reply]
I got a 500 error on reload, then reloaded again and everything was fine; thank god!

**Reply by ilyabo** [reply]
Mine returned, and then disappeared again 🤦

**Comment by DeerWhale** [comment]
Same thing here, all the items disappeared! And only the first time came back, the rest are still gone > <

**Comment by holm10** [comment]
Still experiencing this issue, this is scaring me...

**Comment by fzinnah** [comment]
also having this issue. reloaded a few times after getting 500 error but only newly added issues are showing up.

**Comment by DNin01** [comment]
Related discussion: #67636

I'm sure your issues are coming back as GitHub is [working on restoring **access to Projects data**](https://www.githubstatus.com/incidents/zjchv3zvfg50), implying that nothing was lost, but simply inaccessible.

> We continue working on restoring access to Projects data, along with an estimated time to full restoration.

**Reply by holm10** [reply]
Thanks for linking! Googling 'github projects issue' and finding anything is like looking for a needle in a haystack.

**Comment by rmshaffer** [comment]
Closing as duplicate of #67636

**Reply by ViniciusBatestin** [reply]
All back to normal here. Thank you for manage the topic 🙏
